includes/admin/upgrades/upgrade-functions.php

```diff
@@ -1492,29 +1492,31 @@ function edd_upgrade_render_v30_migration() {
 	<div class="metabox-holder">
 		<div class="postbox">
 			<h2 class="hndle">
-				<span><?php printf( esc_html__( 'Step %d: Upgrade Orders', 'easy-digital-downloads' ), $step ); ?></span>
+				<span>
+					<?php echo esc_html( sprintf( /* translators: %d Migration step count */ __( 'Step %d: Upgrade Orders', 'easy-digital-downloads' ), $step ) ); ?>
```

Comment:
@robincornett The reason is to have an escaping function on the outer most layer of the output. `printf()` will throw a WPCS error as not being escaped. Even though we know it is, it would require a `// WPCS: XSS okay` comment to stop the WPCS sniff from being triggered.